src/GlobalCoders.PSP.BackendApi/SurchargeManagement/Entities/SurchargeEntitiy.cs

```diff
@@ -0,0 +1,38 @@
+using System.ComponentModel.DataAnnotations;
+using GlobalCoders.PSP.BackendApi.SurchargeManagement.Constants;
+
+namespace GlobalCoders.PSP.BackendApi.SurchargeManagement.Entities;
+
+public class SurchargeEntity
+{
+    public Guid Id { get; set; } // Primary Key
+
+    [StringLength(SurchargeConstants.DefaultStringLimitation)]
+    public string Name { get; set; } = String.Empty; // Name of the surcharge
+
+    public decimal Value { get; set; } // Value of the surcharge
+
+    [StringLength(SurchargeConstants.DefaultStringLimitation)]
+    public string Type { get; set; } = String.Empty; // "Percentage" or "Value"
+
+    public DateTime CreationDateTime { get; set; } // When it was created
+
+    [StringLength(SurchargeConstants.DefaultStringLimitation)]
+    public string Status { get; set; } = String.Empty; // "Active" or "Inactive"

```

Comment:
Can you create and use Enum instead of string?
Like: Status{
Active = 1
Inactive = 2
}